test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -2301,13 +2301,14 @@ private static IDisposable TrustPrimaryTimestampRootCertificate(PrimarySignature
         private static IDisposable TrustRootCertificate(IX509CertificateChain certificateChain)
         {
             var rootCertificate = certificateChain.Last();
+            var storeLocation = CertificateStoreUtilities.GetTrustedCertificateStoreLocation();
```

Comment:
Use explicit type (`StoreLocation`) instead of `var`.

Reply:
Thanks! Fixed.